src/vl-tooltip.js

```diff
@@ -1,6 +1,6 @@
 import { VlElement } from '/node_modules/vl-ui-core/dist/vl-core.js';
 import '/node_modules/@govflanders/vl-ui-util/dist/js/util.js';
-import '/node_modules/@govflanders/vl-ui-tooltip/dist/js/tooltip.js';
+import '/lib/tooltip.js';
```

Comment:
hier heb je zelfs iets aan gefixed dan nu?

Reply:
Klopt specifieker:
Van:
``` JavaScript
if (!document.body.contains(_this4._tooltipNode)) {
  return;
} // if we are hiding because of a mouseleave, we must check that the new
// reference isn't the tooltip, because in this case we don't want to hide it
```

Naar:
``` JavaScript
if (!reference.getRootNode().contains(_this4._tooltipNode)) {
  return;
} // if we are hiding because of a mouseleave, we must check that the new
// reference isn't the tooltip, because in this case we don't want to hide it
```

Ik heb ook issue #68 aangemaakt om deze code te pushen naar Webuniversum.

Reply:
De diff:

```3076c3076
<           if (!document.body.contains(_this4._tooltipNode)) {
---
>           if (!reference.getRootNode().contains(_this4._tooltipNode)) {
```

Reply:
@pascalriquier nu kan je kiezen of je mijn of @SanderKleykens zijn comment leest.

Reply:
Ah ziet, je was te snel :)